crates/kubelet/src/pod/mod.rs

```diff
@@ -164,16 +166,11 @@ impl Pod {
     }
 
     /// Gets all of a pod's containers (init and application)
-    pub fn all_containers(&self) -> Vec<ContainerKey> {
-        let app_containers = self.containers();
-        let app_container_keys = app_containers
-            .iter()
-            .map(|c| ContainerKey::App(c.name().to_owned()));
+    pub fn all_containers(&self) -> Vec<Container> {
```

Comment:
This seemed inconsistent with the other two methods `containers` and `init_containers`.

Reply:
I suspect at the time it was written it was used primarily for getting keys/IDs rather than the full container objects. But it looks like now the only places it is used would work just as well with the full container objects.  There is also a place in `store/mod.rs` that explicitly constructs its own `all_containers` collection which this modified method could supplant.